packages/appstore/src/components/modals/modal-manager.tsx

```diff
@@ -126,6 +134,12 @@ const ModalManager = () => {
 
     return (
         <React.Fragment>
+            <WalletsConsentPopup
+                show_wallet_consent_popup={show_wallet_consent_popup}
```

Comment:
all of this props can be handled inside Modal itself instead of passing them here

Reply:
Done 😃 👍🏼